docs/section5/Rule5-7-1.md

```diff
@@ -0,0 +1,36 @@
+# Envelope - Rule 5-7-1 
+**Rule ID:** 5-7-1  
+**Rule Description:** Baseline WWR should be equal to User WWR or 40%, whichever is smaller.  
+**Rule Assertion:** Baseline RMR = expected value  
+**Appendix G Section:** Envelope  
+**Appendix G Section Reference:**
+- Table G3.1, 5. Building Envelope, Baseline Building Performance, c. Vertical Fenestration Areas
+- Table G3.1.1-1
+
+**Data Lookup:** Table G3.1.1-1  
+**Evaluation Context:**  Each Data Element  
+
+**Applicability Checks:** 
+1. Building has spaces that are NEW, ADDITION or ALTERATION.
+
+**Manual Checks:** None  
+
+## Rule Logic:
```

Comment:
In order to disambiguate the space names for reporting purposes, I believe we need to first loop through the buildings in the Baseline RMR.   

For each building in Baseline RMR: `for _building in B_RMR.buildings`